tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -537,16 +545,81 @@ stages:
+              echo "FLUID_TEST_LOGGER_PKG_PATH= ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger"
+              echo "FLUID_BUILD_ID ${{ variables.artifactBuildId }}"
+
+              # Run execution time tests
+              echo "FLUID_ENDPOINTNAME = odsp"
+              npm run test:benchmark:report:odsp;
+              executionTimeTestsExitCodeOdsp=$?;
+
+              # Run memory tests
+              echo "FLUID_ENDPOINTNAME = odsp"
+              npm run test:memory-profiling:report:odsp;
+              memoryTestsExitCodeOdsp=$?;
+
+              if [[ $executionTimeTestsExitCodeOdsp -ne 0 ]]; then
+                echo "##vso[task.logissue type=error]Exit code for runtime tests execution Odsp= $executionTimeTestsExitCodeOdsp"
+              fi
+
+              if [[ $memoryTestsExitCodeOdsp -ne 0 ]]; then
+                echo "##vso[task.logissue type=error]Exit code for memory tests execution Odsp = $memoryTestsExitCodeOdsp"
+              fi
+
+              if [[ $executionTimeTestsExitCodeOdsp -ne 0 ]] ||  [[ $memoryTestsExitCodeOdsp -ne 0 ]]; then
+                exit 1;
+              fi
+          env:
+            login__microsoft__clientId: $(login-microsoft-clientId)
+            login__microsoft__secret: $(login-microsoft-secret)
+            login__odsp__test__tenants: $(automation-stress-login-odsp-test-tenants)
+            FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
+            FLUID_BUILD_ID: ${{ variables.artifactBuildId }}
+            FLUID_ENDPOINTNAME: odsp
+
+        - task: Bash@3
+          displayName: Write measurements to Aria/Kusto - execution time odsp
+          condition: succeededOrFailed()
+          inputs:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              echo "Writing the following performance tests results to Aria/Kusto - ODSP"
+              ls -la ${{ variables.executionTimeTestOutputFolder }};
+              node --require @ff-internal/aria-logger bin/run --handlerModule $(toolAbsolutePath)/dist/handlers/executionTimeTestHandler.js --dir ${{ variables.executionTimeTestOutputFolder }};
+          env:
+            FLUID_ENDPOINTNAME: "odsp"
+
+        - task: Bash@3
+          displayName: Write measurements to Aria/Kusto - memory usage odsp
+          condition: succeededOrFailed()
+          inputs:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              echo "Writing the following performance tests results to Aria/Kusto - ODSP"
+              ls -la ${{ variables.memoryUsageTestOutputFolder }};
+              node --require @ff-internal/aria-logger bin/run --handlerModule $(toolAbsolutePath)/dist/handlers/memoryUsageTestHandler.js --dir ${{ variables.memoryUsageTestOutputFolder }};
+          env:
+            FLUID_ENDPOINTNAME: "odsp"
 
```

Comment:
In  this newly added pipeline step you are only submitting the metrics to Aria / Kusto. I think that is actually preferrable for now so I can confirm the use case with my team next week. If they are good with it (and you are as well) then we can add the additional step to push that to App Insights as well